### PR TITLE
LPS-128643 Delete actions looks bad in KB

### PR DESCRIPTION
**Pull request by boton**
Hi guys, 
I try to avoid the render of `<liferay-ui:icon-delete` with label and icon inside a small button with no space.
I try only adding the `component-action` CSS class when the tag doesn't have a label.

**How to test it:**

1. Go to add a page with the knowledge base display widget
2. Go add a KB article
3. Go to the page to view

**Before the fix**
image

**After the fix**
image

### CODE REVIEW

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

File: util-taglib/src/com/liferay/taglib/ui/IconDeleteTag.java [hunk 1]
```diff
@@ -91,7 +91,9 @@ protected String getPage() {
 				}
 			}
 
-			setLinkCssClass("component-action");
+			if (!isLabel()) {
+				setLinkCssClass("component-action");
```
Comment (diegonvs) [inline util-taglib/src/com/liferay/taglib/ui/IconDeleteTag.java:95]: Looks good to me! 

When taking a look on Clay documentation, I found a very short description for `component-action`:

> Use these patterns for actions in components.

Are "actions" intended to not have text content inside?

Source: https://clayui.com/docs/components/link/markup.html#css-component-action

@matuzalemsteles @pat270 @julien? I can open an issue for a more detailed description :)

**Comment by boton** [comment]
ci:test:relevant

**Comment by boton** [comment]
ci:test:relevant

**Comment by boton** [comment]
ci:test:relevant

File: util-taglib/src/com/liferay/taglib/ui/IconDeleteTag.java (hunk 1 above)
Comment (matuzalemsteles) [reply util-taglib/src/com/liferay/taglib/ui/IconDeleteTag.java:95]: I think the `component-action` is not necessarily just for text but an button with an icon is also considered an action, @pat270 could explain more.

Comment (julien) [reply util-taglib/src/com/liferay/taglib/ui/IconDeleteTag.java:95]: @diegonvs good question: I have no idea of the reply. Maybe @drakonux and has the answer?
I'm going to have a look at the lexicon website to see if I can find more information.

**Comment by julien** [comment]
@boton not sure if this is rebased with master, the CI errors don't look related.
If you need help identifying them, don't hesitate to ask @john-co for help.

**Comment by boton** [comment]
Now, it's rebased with master again :D

**Comment by boton** [comment]
ci:test:relevant

**Comment by boton** [comment]
This semantic error version doesn't look related to my changes

```
AXIS_VARIABLE=0,label_exp=!master #387147
Please fix semantic versioning on boton/LPS-128643-bug-MB-wrong-styles-delete-link
```

We will see after the rebase

**Review by julien — APPROVED** [review]

**Comment by julien** [comment]
ci:forward

**Pull request closed without merging**